base/setup/usetup/usetup.c

```diff
@@ -626,6 +626,171 @@ UpdateKBLayout(VOID)
     }
 }
 
+/*
+ * Function Name: ProgressTimeOutStringHandler
+ *
+ * Purpose: Handles the generation (displaying) of the timeout countdown to the screen
+ *          dinamically
+ *
+ * Return: FALSE
+ */
+static
+BOOLEAN NTAPI
+ProgressTimeOutStringHandler(
+    IN PPROGRESSBAR Bar,
+    IN BOOLEAN AlwaysUpdate,
+    OUT PCHAR Buffer,
```

Comment:
Please revert the PCHAR back into PSTR since the string that is placed in this buffer will always be NULL-terminated (i.e. @ThFabba was not correct), and this is expected by the caller of this function.